include/mgba/internal/arm/decoder.h

```diff
@@ -11,6 +11,7 @@
 CXX_GUARD_START
 
 #include <mgba/internal/arm/arm.h>
+#include <mgba/debugger/debugger.h>
```

Comment:
```suggestion
#include <mgba/debugger/debugger.h>
#include <mgba/internal/arm/arm.h>
```

Keep these in alphabetical order.